inc/Engine/Optimization/DeferJS/AdminSubscriber.php

```diff
@@ -44,4 +45,22 @@ public static function get_subscribed_events() : array {
 	public function add_defer_js_option( array $options ) : array {
 		return $this->defer_js->add_option( $options );
 	}
+
+	/**
+	 * Adds jQuery to defer JS exclusion field if safe mode was enabled before 3.8
+	 *
+	 * @since 3.7
```

Comment:
I think this should be 3.8, right?

```suggestion
	 * @since 3.8
```